apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayout.java

```diff
@@ -128,6 +128,11 @@ public static RifLayout parse(Workbook workbook, String sheetName) {
       if ("HHA".contains(sheetName) && "CLM_HHA_TOT_VISIT_CNT".equals(rifColumnName))
         rifColumnLength = Optional.of(4);
 
+      // Temporary Code for CCW Additions. These will go away
+      if ("HHA,Hospice".contains(sheetName) && "CLAIM_QUERY_CODE".equals(rifColumnName)) continue;
+      // Temporary Code for CCW Additions. These will go away
```

Comment:
So not sure why we are still commenting these out; my understanding is that all these 'sub-tickets' will fall under a parent ticket which encompasses all the CCW changes. Thus, your changes will simple be merged into the 'parent' branch, which will eventually be merged into master. So I don't believe we need to worry about doing things like ignoring some CCW fields taht we know will eventually be part of BFD.